pipeline/configs/ami-ec2-instance-props.json

```diff
@@ -1,6 +1,6 @@
 {
   "InstanceType": "c5.xlarge",
-  "ImageName": "amzn-ami-2017.09.d-amazon-ecs-optimized",
+  "ImageName": "amzn-ami-2018.03.20200115-amazon-ecs-optimized",

```

Comment:
the old (2017) AMI has an outdated [ECS Container Agent](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_agent.html). As of 24 Jan, 2020, the updated (2018) AMI has the current agent: version 1.36.1. However, a longer term fix might be to make note in the deployment script to check for a new version and update the json file before attempting a deployment. Further, it might be helpful to include steps to swap out the AMI when the current one becomes outdated.